styles/variables.css

```
@@ -0,0 +1,32 @@
+:root {
+  /* ----- typography ----- */
+  --h1-font-size: 4rem;
+  --h2-font-size: 3rem;
+  --h3-font-size: 2.25rem;
+  --h4-font-size: 1.5rem;
+  --paragraph-font-size: 1.25rem;
+
+  /* ----- line-hight ----- */
```

```suggestion
  /* ----- line-height ----- */
```
line height should be default one - let's leave it as is and see if we'll need it 😉